recipes-bsp/u-boot/files/upstream/0001-arm-dts-Add-IOT2050-device-tree-files.patch

```diff
@@ -545,6 +520,7 @@ index 0000000000..fcb45d1525
 +		&push_button_pins_default
 +		&db9_com_mode_pins_default
```

Comment:
There is no related function about DB9 Port under uboot. Should we remove these nodes?

Reply:
The idea is to use the kernel DT, just plus additional properties that U-Boot needs. Makes maintenance easier and was actually requested.